arcgis-ios-sdk-samples/Maps/Show device location using indoor positioning/ShowDeviceLocationUsingIndoorPositioning.storyboard

```diff
@@ -0,0 +1,84 @@
+                                <rect key="frame" x="0.0" y="0.0" width="600" height="41"/>
+                                <subviews>
+                                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="0nw-hp-v4a">
+                                        <rect key="frame" x="20" y="0.0" width="560" height="41"/>
+                                        <subviews>
+                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Location info will be shown here." lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="KPY-5n-i0P">
+                                                <rect key="frame" x="0.0" y="0.0" width="560" height="20.5"/>
+                                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                <nil key="textColor"/>
+                                                <nil key="highlightedColor"/>
+                                            </label>
+                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Transmitter info will be shown here." lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="7rI-gP-uHy">
+                                                <rect key="frame" x="0.0" y="20.5" width="560" height="20.5"/>
+                                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                <nil key="textColor"/>
+                                                <nil key="highlightedColor"/>
+                                            </label>
+                                        </subviews>
+                                    </stackView>
+                                </subviews>
+                                <viewLayoutGuide key="safeArea" id="3n2-oF-hrI"/>
+                                <color key="backgroundColor" name="statusLabelBackgroundColor"/>
+                                <constraints>
+                                    <constraint firstAttribute="trailing" secondItem="0nw-hp-v4a" secondAttribute="trailing" constant="20" symbolic="YES" id="Cea-ld-jRc"/>
+                                    <constraint firstItem="0nw-hp-v4a" firstAttribute="leading" secondItem="Qsl-Ei-BAa" secondAttribute="leading" constant="20" symbolic="YES" id="I6K-3l-m43"/>
```

Comment:
This leading edge of the stack view should be constrained to the leading margin of the super view (which includes the safe area) so that the indentation will be correct. It should line up with the back button and the text in the attribution bar:

Leading | Leading Margin
--- | ---
![image](https://user-images.githubusercontent.com/2257493/155492165-517afa9e-4c78-48fe-b68a-2edab112dcf0.png) | ![image](https://user-images.githubusercontent.com/2257493/155493172-bb58bda0-1cf1-4828-b659-70cc1c59c4c1.png)

That's better, although it still isn't quite right--now it is indented slightly too far. I think that is due to the default layout margins of the stack view. But it doesn't need to be perfect.

Same with the trailing constraint.